_drafts/2017-xx-xx-patentsview.Rmd

```diff
@@ -0,0 +1,244 @@
+---
+title: "Accessing patent data with the patentsview package"
+name: patentsview
+author:
+  - name: Chris Baker
+    url: http://github.com/crew102
+date: '2017-08-20'
+layout: post
+tags:
+  - R
+  - Patents
+  - PatentsView
+  - Package
+  - API
+  - USPTO
+categories: blog
+---
+
+```{r, echo = FALSE, message = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  warning = FALSE,
+  message = FALSE
+)
+```
+
+## Why care about patents?
+
+**1. Patents play a critical role in incentivizing innovation, without which we wouldn't have much of the technology we rely on everyday**
+
+What does the your iPhone, Google's PageRank algorithm, and a butter substitute called [Smart Balance](https://www.smartbalance.com/) all have in common? 
+
+<span> <!-- These are open source images taken from: https://pixabay.com/ -->
+<img src="2017-xx-xx-patentsview-iphone.png" width="15%">
+<img src="2017-xx-xx-patentsview-google.jpg" width="25%">
+<img src="2017-xx-xx-patentsview-butter.png" width="25%">
+</span>
+
+...They all probably wouldn't be here if not for patents. A patent provides its owner with the ability to make money off of something that they invented, without having to worry about someone else copying their technology. Think Apple would spend millions of dollars developing the iPhone if Samsung could just come along and [rip it off](http://www.reuters.com/article/us-apple-samsung-elec-appeal-idUSKCN1271LF)? Probably not. 
+
+**2. Patents offer a great opportunity for data analysis**
+
+There are two primary reasons for this:
+
+  - **Patent data is public**. In return for the exclusive right to profit off an invention, an individual/company has to publicly disclose the details of their invention to the rest of the world. Those details come in the form of a patent's title, abstract, technology classification, assigned organizations, etc.
+  - **Patent data can answer questions that people care about**. Companies (especially big ones like IBM and Google) have a vested interest in extracting insights from patents, and spend a lot of time/resources trying figure out how to best manage their intellectual property (IP) rights.  They're plagued by questions like "who should i sell my underperforming patents to," "which technology areas are open to new innovations", "what's going to be the next big thing in the world of buttery spreads," etc. Patents offer a way to provide data-driven answers to these questions.
```

Comment:
Would you mind if we go with the US standard of putting commas/periods inside quotes instead of outside? I know that some countries put them on the outside, but I prefer them on the inside (though it looks like I mistakenly put one comma on the outside by accident).